frontend/packages/dev-console/src/components/import/__mocks__/deployImage-validation-mock.ts

```diff
@@ -56,14 +56,14 @@ export const mockFormData: DeployImageFormData = {
     },
     name: 'myimage',
     tag: 'latest',
-    status: 'success',
+    status: { metadata: {}, status: 'success' },
```

Comment:
The bad type for status was used by a previous PR, so I had to update it once I rebased - didn't catch the typescript error when I rebased on master.